sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Demo/Program.cs

```diff
@@ -15,7 +15,7 @@
 /*
 builder.Services.AddAzureMonitor(o =>
 {
-    o.ConnectionString = "InstrumentationKey=00000000-0000-0000-0000-000000000000";
+    o.ConnectionString = "InstrumentationKey=00000000-0000-0000-0000-00000000CODE";
```

Comment:
any particular reason to change this?

Reply:
I see the change below.

Reply:
makes it easier to understand, as we have entry in both file and appSettings. Personally, felt 123 is not a good.